src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -1047,6 +1059,19 @@ vcuKernelTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tre
   runTestCase(_dev, "xcl_vcu_test.exe", _ptTest.get<std::string>("xclbin"), _ptTest);
 }
 
+/*
+ * TEST #12
+ */
+void
+iopsTest(const std::shared_ptr<xrt_core::device>& _dev, boost::property_tree::ptree& _ptTest)
+{
+  if(!search_and_program_xclbin(_dev, _ptTest)) {
+    return;
+  }
```

Comment:
This isn't needed. `runTestCase` should find the xclbin and `runScript` should program it